pkgs/by-name/an/anakron/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  xorg,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "anakron";
+  version = "0.3.1";
+
+  src = fetchzip {
+    url = "https://github.com/molarmanful/ANAKRON/releases/download/v${version}/ANAKRON-release_v${version}.zip";
```

Comment:
why not fetchFromGitHub?

Reply:
It's a good question! I've looked through some other bitmap fonts and they use `fetchzip`. I've decided to stick to their approach 🤷

Since it's not a good reason I'm going to update this package to use `fetchFromGitHub`.

Reply:
I've looked into this a little bit more. There is a reason!

With `fetchzip` I'm downloading an artifact that is built by a CI system, not the zip archive of the source code.

`fetchFromGitHub` is limited to downloading the source code.

Reply:
Makes sense. Thanks for explaining it to me!